source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRViewController.h

```diff
@@ -7,10 +7,13 @@
 
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
+#import "ACRActionDelegate.h"
 
 @interface ACRViewController :UIViewController
 
 @property NSString* jsonString;
+@property id<ACRActionDelegate> acrActionDelegate;
```

Comment:
>property [](start = 1, length = 8)

This property should probably be weak, or else there's likely to be a retain cycle between it and the view controller.